src/main/java/com/minecolonies/colony/permissions/Permissions.java

```diff
@@ -464,6 +464,24 @@ public UUID getOwner()
     }
 
     /**
+     * Returns the name of the owner of this permission instance.
+     *
+     * @return Name of the owner
+     */
+    @Nullable
+    public String getOwnerName()
+    {
+        for (@NotNull final Map.Entry<UUID, Player> entry : players.entrySet())
```

Comment:
Relating to the above comment, if we saved the owner UUID, this method would be just players.get(ownerUUID).getName().